README.md

```diff
@@ -585,6 +585,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [loadmill.com](https://www.loadmill.com/) - Automatically create API and load tests by analyzing network traffic. Simulate up to 50 concurrent users for up to 60 minutes for free every month.
   * [preflight.com](https://preflight.com) - No-code automated web testing. Record tests on your browser that are resilient to UI changes and run them on Windows machines. Integrate with your CI/CD. The free plan includes 50 test runs per month with video, HTML sessions and more.
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  Unlimited team members, Demo app and unlimited projects, 5,000 snapshots / month.
+  * [Playwright](https://playwright.dev) - Batteries-included test automation framework for end-to-end testing. Its cross-platform, cross-browser, and cross-language, and includes helpful features like auto-waiting, web-first assertions, tracing, parallel test execution, HTML report and etc. Playwright is built to enable cross-browser web automation that is ever-green, capable, reliable and fast.
```

Comment:
This is a list for SaaS software, please describe what is offered as a SaaS.

Reply:
Sorry, my bad. I didn't read beyond the statement "free for dev".